mod-inn-reach/src/main/resources/volaris/mod-inn-reach/global/mod-item-util.feature

```diff
@@ -0,0 +1,29 @@
+Feature: setup user data feature
```

Comment:
there is already a feature to create inventory data -`mod_inventory_init_data.feature` where the instance is being created, just update it to create holding and item as well.